vendor/assets/javascripts/jquery.textile.js

```diff
@@ -27,7 +27,7 @@
         resize: true,
         // HTML templates
         tpl: {
-          wrap: '<div class="textile-wrap"><ul class="textile-toolbar"></ul><div class="textile-inner row"></div></div>',
+          wrap: '<div class="textile-wrap"><ul class="textile-toolbar"></ul><div class="textile-inner row" data-behavior="textile-inner"></div></div>',
```

Comment:
@MattBudz is "textile-inner" really a behaviour. As far as semantics go it doesn't tell me what the _behaviour_ *is*. I know we're clearly binding it to something but I wouldn't know what.